ale_linters/html/tidy.vim

```diff
@@ -2,53 +2,79 @@
 " Description: This file adds support for checking HTML code with tidy.
 
 if exists('g:loaded_ale_linters_html_tidy')
-    finish
+	finish
```

Comment:
I think there are perhaps tabs throughout the file. It's easier for me to tell in Vim itself.